include/yaramod/types/string.h

```diff
@@ -245,6 +253,17 @@ class String
 		_mods[modifier->getType()] = modifier;
 		return true;
 	}
+
+	void removeModifiers()
+	{
+		for (const auto& mod : _mods)
```

Comment:
Just a heads up for the future, there are now [structured bindings](https://en.cppreference.com/w/cpp/language/structured_binding) since C++17 which make working with pairs/tuples much better.